jax/_src/maps.py

```diff
@@ -285,6 +285,13 @@ def xmap(fun: Callable,
          backend: str | None = None) -> stages.Wrapped:
   """Assign a positional signature to a program that uses named array axes.
 
+  .. warning::
+    xmap is deprecated and will be removed in a future release. Use
+    :py:func:`~jax.shard_map` or :py:func:`~jax.vmap` with the
```

Comment:
This is still `jax.experimental.shard_map`, yes?

Reply:
It's worse! It's `jax.experimental.shard_map.shard_map` :)

Should I use that and add a comment on #20124 to replace it with `jax.shard_map`?

Reply:
Well, if #20124 is going in soon then we can probably leave this as-is.